gateway/redis_signals.go

```diff
@@ -35,6 +35,7 @@ const (
 	NoticeGatewayDRLNotification NotificationCommand = "NoticeGatewayDRLNotification"
 	KeySpaceUpdateNotification   NotificationCommand = "KeySpaceUpdateNotification"
 	OAuthPurgeLapsedTokens       NotificationCommand = "OAuthPurgeLapsedTokens"
+	noticeDeleteAPICache                             = NotificationCommand(deleteAPICache)
```

Comment:
Follow style above this line

Reply:
is it about not exporting or the naming?

Reply:
It's about omitting the type and not using the literal value, the addition deviates from the style above; suggesting to remove deleteApiCache symbol ; as for exporting the var, asume export=yes as it is a typed value